woocommerce/woocommerce-android
# Themes: Theme carousel

Fixes #10234. The PR adds the theme picker screen with the theme carousel.

<image src="https://github.com/woocommerce/woocommerce-android/assets/1522856/51fdefc9-8178-4513-901c-ca25cba6dd6d" width="300" />

**To test:**
1. Start the app and trigger the store creation
2. Go through the profiler steps
3. After the feature profiler screen, notice the next screen is the theme picker screen (only when feature flag is enabled)
4. Notice a couple of demo site screenshots are loaded (some are not working, they're just placeholders)
